processing-and-analysis/02-command-files/Ganling's file/01-Model_Analysis_New_Data.Rmd

```diff
@@ -0,0 +1,317 @@
+                 satm_c + satv_c + aleksikc_c + hs_gpa_c, 
+               data = df_true)
+AIC(sex_mod1, sex_mod3)
+summary(sex_mod3)
+```
+* The AIC value decreased for `sex_mod3`
+* The interaction between `exp` and `sex_id` should not be included
+
+#### Sex model 3.a: No `exp` 
+```{r}
+sex_mod3.a <- lm(final_c ~ sex_id + 
+                 satm_c + satv_c + aleksikc_c + hs_gpa_c, 
+               data = df_true)
+AIC(sex_mod3, sex_mod3.a)
+summary(sex_mod3.a)
+```
+* The AIC value slightly decreased
+* Since sex_mod3.a is slightly smaller, we keep this model
+
+#### Sex model 3.b: Remove `sex_id`
+```{r}
+sex_mod3.b <- lm(final_c ~ satm_c + satv_c + aleksikc_c + hs_gpa_c, 
+               data = df_true)
+AIC(sex_mod3.a, sex_mod3.b)
+summary(sex_mod3.b)
+```
+* Since the AIC value increased, `sex_id` should be kept in the model
+
+#### Sex model 3.c: Remove `satv_c`
+```{r}
+sex_mod3.c <- lm(final_c ~ sex_id + 
+                 satm_c + aleksikc_c + hs_gpa_c, 
+               data = df_true)
+AIC(sex_mod3.a, sex_mod3.c)
+summary(sex_mod3.c)
+```
+* The AIC value went up, so `satv_c` should be kept in our model 
+
+#### Sex model 3.d: Remove `aleksikc_c`
+```{r}
+sex_mod3.d <- lm(final_c ~ sex_id + 
+                 satm_c + satv_c + hs_gpa_c, 
+               data = df_true)
+AIC(sex_mod3.a, sex_mod3.d)
+summary(sex_mod3.d)
+```
+* The AIC value increased, `aleksikc_c` should be kept
+
+#### Sex Model 3.e: remove `hs_gpa_c` 
+```{r}
+sex_mod3.e <- lm(final_c ~ sex_id + 
+                 satm_c + satv_c + aleksikc_c , 
+               data = df_true)
+AIC(sex_mod3.a, sex_mod3.e)
+summary(sex_mod3.e)
+```
+* The AIC value went up, `hs_gpa_c` is kept
+
+#### Sex Model 3.f: Remove `satm_c`
+```{r}
+sex_mod3.f <- lm(final_c ~ sex_id + 
+                 satv_c + aleksikc_c + hs_gpa_c, 
+               data = df_true)
+AIC(sex_mod3.a, sex_mod3.f)
+summary(sex_mod3.f)
+```
+* Again, the AIC value went up. `satm_c` should be kept
+* `sex_mod3.a` is the best fitting model  
+
+### Model selection on `eop_id`
+#### Eop model 1: fixed effect
+```{r}
+eop_mod1 <- lm(final_c ~ exp * eop_id + 
+                 exp + eop_id + 
+                 satm_c + satv_c + aleksikc_c + hs_gpa_c, 
+               data = df_true)
+summary(eop_mod1)
+```
+#### EOP model 2: random effect
+```{r}
+eop_mod2 <- lmer(final_c ~ exp * eop_id + 
+                   exp + eop_id +
+                   satm_c + satv_c + aleksikc_c + hs_gpa_c + 
+                   (1 | ta_sect),
+                 data = df_true, REML = TRUE)
+summary(eop_mod2)
+AIC(eop_mod1, eop_mod2)
+```
+* AIC value went up, random effect should not be removed
+* Interaction has the lowest t value, it should be removed next
+
+#### Eop model 3: remove `exp` and `eop_id` interaction and no radnom effect 
+```{r}
+eop_mod3 <- lm(final_c ~ exp + eop_id + 
+                 satm_c + satv_c + aleksikc_c + hs_gpa_c, 
+               data = df_true)
+AIC(eop_mod1, eop_mod3)
+summary(eop_mod3)
```

Comment:
Should be named `eop_mod1.a`, per step 5 in [model selection guide](https://docs.google.com/document/d/1XRv2093XdzpSqRVzrrusUUVNMunE6scFvDH-kqAPnPU/edit#heading=h.fa5ifiscpnpb)